apps/src/gamelab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -53,7 +52,7 @@ const styles = {
 
 const AnimationPickerListItem = React.createClass({
   propTypes: {
-    animation: React.PropTypes.shape(METADATA_SHAPE),
+    animation: React.PropTypes.object, // TODO: Shape?
```

Comment:
Okay to leave for now.  These shapes are still somewhat in flux (see the many type definitions at the bottom of animationListModule.js).  It'll also be nice to do any renames within the animation redux structure before we apply these shapes everywhere.